<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureWidget</class>
 <widget class="QWidget" name="MeasureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeasureWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="idTitle1">
     <property name="text">
      <string>Image id:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="slider1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QSlider" name="slider2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="idTitle2">
     <property name="text">
      <string>Image id:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QLabel" name="image2">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="id1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="id2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="image1">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="distance">
     <property name="text">
      <string>Distance:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="status1">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="pos2d1">
     <property name="text">
      <string>Position in 2D:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="pos3d1">
     <property name="text">
      <string>Position in 3D:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QLabel" name="status2">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="QLabel" name="pos2d2">
     <property name="text">
      <string>Position in 2D:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QLabel" name="pos3d2">
     <property name="text">
      <string>Position in 3D:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="5" margin="0"/>
 <resources/>
 <connections/>
</ui>
